### ISSUE TITLE
token process

### ISSUE THREAD

**Issue by Repair-Desk** [iZettle/sdk-ios#197]
We require the merchant initialization function. We did not found any option for initialization except that it is taking email or password by merchant.

We are talking about token that is received from server.

**Comment by arikis** [comment]
Hey @Repair-Desk, just to clarify, are you talking about OAuth token or some other token?

**Comment by Repair-Desk** [comment]
Thanks for the message, we'll get back to you as soon as
possible!

On Apr 16, 2018 at 12:53 PM iZettle/sdk-ios wrote:

Hey @Repair-Desk, just to clarify, are you talking about OAuth
token or some other token?

--

You are receiving this because you were mentioned.

Reply to this email directly or view it on GitHub:

https://github.com/iZettle/sdk-ios/issues/197#issuecomment-381727930
( https://github.com/iZettle/sdk-ios/issues/197#issuecomment-381727930 )

Powered by UserVoice. ( http://www.uservoice.com/powered-by?utm_campaign=poweredby&utm_medium=email&utm_source=ticket&uv_company=RepairDesk&uv_contact_url=repairdesk.uservoice.com&uv_experience=email )

**Comment by Repair-Desk** [comment]
Yes, we are talking about oauth token process and kindly elaborate  function name that how to call it  in swift or objective c
